buildmaster/LHCB_Z_13TEV_DIMUON_2022/metadata.yaml

```
@@ -0,0 +1,104 @@
+setname: "LHCB_Z_13TEV_DIMUON_2022"
+
+nnpdf_metadata:
+  nnpdf31_process: "Z0PT"
```

```suggestion
  nnpdf31_process: "NC DY"
```

This is probably what you want here. But check the other `Z0` and `Z0J` datasets to check what to write here.
This information is used for the theory covariance matrix so it is important that it matches between datasets.

---

I just checked [ATLAS_Z0J_8TEV](https://github.com/NNPDF/nnpdf/blob/master/validphys2/src/validphys/datafiles/new_commondata/ATLAS_Z0J_8TEV/metadata.yaml) and [LHCB_Z0_13TEV](https://github.com/NNPDF/nnpdf/blob/master/validphys2/src/validphys/datafiles/new_commondata/LHCB_Z0_13TEV/metadata.yaml) and it seems that `nnpdf31_process` is `DY NC` for both `Z0` and `Z0J`.

Anyway, I appreciate that you also provide an explanation for what these values actually do.